### ISSUE TITLE
Big Picture feature request: show FPS

### ISSUE THREAD

**Issue by adolson** [ValveSoftware/steam-for-linux#4115]
Please add an option to Big Picture, perhaps in the overlay, to enable the FPS counter like the desktop mode users have.

**Comment by liamdawe** [comment]
Yes this please, for SteamOS too not sure why it has been left out, but it's an extremely useful feature of Steam.

Particularly useful for people testing different hardware with SteamOS like me.

**Comment by tiberiusferreira** [comment]
I was a bit surprised it was not implemented yet.

**Comment by Rodvil** [comment]
I had my voice to this request. It's one of the things preventing me from using big picture more often

**Comment by Grief** [comment]
This is not implemented for windows as well

**Comment by Sighery** [comment]
Is there any update on this feature request? I use Steam Big Picture quite often to play from the couch and the only thing I'm missing is the FPS counter the normal desktop overlay offers. Is there anything us users can do to help get this added?

**Comment by 13r** [comment]
Still waiting for it.

**Comment by adolson** [comment]
I've given up on it ever happening. MangoHud is pretty easy to implement, though, so I use that.

**Comment by Managor** [comment]
Modern Big Picture Mode has an ability to add an FPS counter through the Quick Access Menu,
